### ISSUE TITLE
Error due to Questie's new module loading

### ISSUE THREAD

**Issue by Muehe** [butter-cookie-kitkat/ButterQuestTracker#80]
## Bug description

Hey there, I'm one of the Questie devs and we are getting reports due to BQT throwing the following error ([full trace](https://cdn.discordapp.com/attachments/579966856085897216/644130924765642792/debug.txt)):

```
Date: 2019-11-13 06:00:17
ID: 1
Error occured in: Global
Count: 1
Message: ...tterQuestTracker\LibQuestHelpers\LibQuestHelpers.lua line 125:
   attempt to index global 'QuestieDB' (a nil value)
```

This happens because we recently changed the way Questie modules are loaded in an effort to pollute the global namespace less. You can see an example [here](https://github.com/AeroScripts/QuestieDev/blob/master/Questie.lua#L31).

Now I tried just adding the line for `QuestieDB` in the file mentioned in the error, but it didn't work because `QuestieLoader` wasn't present yet. I guess you would have to wait for Questie's addon loaded event and then import the modules you use.

**Comment by cecilia-sanare** [comment]
Hey @Muehe thanks for the heads up. Looking into this now.

**Comment by cecilia-sanare** [comment]
This issue should now be resolved in [1.9.14](https://www.curseforge.com/wow/addons/butter-quest-tracker/files/2827005).

Thanks again for submitting an issue over here and let me know if your users run into any more problems!
